django/db/models/query.py

```diff
@@ -760,6 +771,7 @@ def bulk_create(
         update_conflicts=False,
         update_fields=None,
         unique_fields=None,
+        condition=None,
```

Comment:
Signature of `abulk_create()` should also be changed (here and in docs).

Reply:
> Signature of `abulk_create()` should also be changed (here and in docs).

why? I didn't work on `abulk_create` but rather on `bulk_create`.

Reply:
> > Signature of `abulk_create()` should also be changed (here and in docs).
> 
> why? I didn't work on `abulk_create` but rather on `bulk_create`.

But you should work on it. `abulk_create()` is an async variant of `bulk_create()` they must provide the same functionality.

Reply:
> > > Signature of `abulk_create()` should also be changed (here and in docs).
> > 
> > 
> > why? I didn't work on `abulk_create` but rather on `bulk_create`.
> 
> But you should work on it. `abulk_create()` is an async variant of `bulk_create()` they must provide the same functionality.

it was not mentioned either in the tiket [#34277 ](https://code.djangoproject.com/ticket/34277#comment:9)or in the previous [#31685](https://code.djangoproject.com/ticket/31685). i also did not see them in @ChihSeanHsu's work [here](https://github.com/django/django/pull/13065)

Reply:
did `abulk_create` take care of @ChihSeanHsu''s previous work?

Reply:
I'm talking to you through a translator, so some of my sentences may be mistranslated.

Reply:
i've just seen the code for `abulk_create`(). it's just the parameter that needs to be added, apparently.
```
async def abulk_create(
        self,
        objs,
        batch_size=None,
        ignore_conflicts=False,
        update_conflicts=False,
        update_fields=None,
        unique_fields=None,
    ):
        return await sync_to_async(self.bulk_create)(
            objs=objs,
            batch_size=batch_size,
            ignore_conflicts=ignore_conflicts,
            update_conflicts=update_conflicts,
            update_fields=update_fields,
            unique_fields=unique_fields,
        )
```